tsconfig.json

```diff
@@ -1,10 +1,15 @@
 {
   "files": [
     "./x-element.js",
+    "./x-template.js",
     "./etc/ready.js"
   ],
   "compilerOptions": {
     "target": "ES2023",
+    // Beyond just organization, a “bug” in how JSDocs works causes a cyclic
```

Comment:
This was a cute one. It’s not really a _bug_ from the perspective of the maintainers. Not terribly hard to workaround though — we just throw things into a `/ts` directory. Maybe we should have just done that from the start 🤷 

Without this… you could build the types _exactly once_. Repeat builds would confuse TypeScript since the filename prefixes are the same. I.e., it would think we were trying to import our export 🤖